src/data-catalog/DataBrowser.test.ts

```diff
@@ -1,7 +1,7 @@
 import * as _ from 'lodash/fp';
-import { convertColsToSettings, convertSettingsToCols, extractCatalogFilters } from 'src/pages/library/DataBrowser';
-import { datasetAccessTypes } from 'src/pages/library/dataBrowser-utils';
-import { TEST_DATASET_ONE, TEST_DATASET_TWO, TEST_DATASETS } from 'src/pages/library/test-datasets';
+import { datasetAccessTypes } from 'src/data-catalog/data-browser-utils';
```

Comment:
I'd suggest considering relative imports within src/data-catalog. Eventually, we'd like to move toward each feature area being a package. Relative imports prepares for that and makes it easier to see where the dependencies between feature areas and on common components are.